translations/pt-br/README.md

```diff
@@ -0,0 +1,175 @@
+Artigos que podem ajudar pessoas a falar em público
+
+Esse repositório contém conteúdo para melhoria contínua e evolução para falar em público. O primeiro foco é apresentações em eventos técnicos, mas muitas dicas poderão ser usadas de diferentes formas para falar em público em eventos e audiências.
+
+* [Contribuindo](#contribuindo)
+* [Recursos](#recursos)
+* [História e Base de conhecimento](#historia)
+* [Licença](#licença)
+
+## Contribuindo
+
+Conhece algum artigo sobre falar em público que você gostaria de compartilhar conosco? 
+
+Por favor, dê uma olhada em [CONTRIBUTING.md](./CONTRIBUTING.md) para saber como contribuir :-)
+
+Todos os contribuidores e participantes deste projeto (incluindo o [canal no IRC](https://webchat.freenode.net/?channels=%23public_speaking)[en]) aceitam em seguir o [Código de Conduta](./CODEOFCONDUCT.md).
+
+Por favor [fale conosco](https://github.com/airtonzanon/Public_Speaking/blob/master/CONTRIBUTING.md#for-more-help) se você tiver qualquer pergunta sobre o projeto ou como contribuir.
+
+## Recursos
+
+Dê uma olhada abaixo ou clique aqui para ir direto na sessão:
+
+* [Geral](#geral)
```

Comment:
Each of these should link back to the main README anchors instead of this doc. For now, we're going to include all of the resources there no matter which language they're in.